src/site/content/en/blog/webcodecs/index.md

```diff
@@ -317,64 +346,44 @@ async function render_frame() {
   let frame = ready_frames.shift();
   underflow = false;
 
-  let bitmap = await frame.createImageBitmap();
   // Based on the frame's timestamp calculate how much of real time waiting
   // is needed before showing the next frame.
   let time_till_next_frame = calculateTimeTillNextFrame(frame.timestamp);
   await delay(time_till_next_frame);
-  ctx.drawImage(bitmap, 0, 0);
+  ctx.drawImage(frame, 0, 0);
+  frame.close();
 
   // Immediately schedule rendering of the next frame
   setTimeout(render_frame, 0);
-  frame.destroy();
 }
 ```
 
 ## Demo
 
-The demo below shows two canvases, the first one is animated at the refresh rate
-of your display, the second one shows a sequence of frames captured by
-`VideoTrackReader` at 30&nbsp;FPS, encoded  and decoded using WebCodecs API.
+The demo below shows how animation frames from a canvas are
+- captured at 25fps into a `ReadableStream` by `MediaStreamTrackProcessor`
+- transferred to a web worker
+- encoded into H.264 video format
+- decoded again into a sequence of video frames
+- and rendered on the second canvas using `transferControlToOffscreen()`
 
-{% Glitch 'webcodecs-blogpost-demo' %}
+{% Glitch 'new-webcodecs-blogpost-demo' %}
+
+## Using the WebCodecs API {: #use }
 
 ## Feature detection
 
 To check for WebCodecs support:
 
 ```js
-if ('VideoEncoder' in window) {
+if ('VideoFrame' in window) {
   // WebCodecs API is supported.
 }
 ```
 
-## Using the WebCodecs API {: #use }
-
-### Enabling via a command line flag
-
-To experiment with the WebCodecs API locally on all desktop platforms, without an
-origin trial token, start Chrome with a command line flag:
-
-```bash
---enable-blink-features=WebCodecs
-```
-
-### Enabling support during the origin trial phase
-
-The WebCodecs API is available on all desktop platforms (Chrome OS, Linux, macOS,
-and Windows) as an origin trial in Chrome&nbsp;86. The origin trial is expected
-to end just before Chrome&nbsp;88 moves to stable in February 2021. The API can
-also be enabled using a flag.
-
-{% include 'content/origin-trials.njk' %}
-
-### Register for the origin trial {: #ot }
-
-{% include 'content/origin-trial-register.njk' %}
-
 ## Feedback {: #feedback }
 
-The Chrome team wants to hear about your experiences with the Idle Detection API.
+The Chrome team wants to hear about your experiences with the WebCodecs API.
 
 ### Tell us about the API design
```

Comment:
I don't think we're soliciting feedback on the overall design any more, we're on to v2 feature requests now.

Reply:
It's predefined boilerplate text from web.dev